core/variant/variant_call.cpp

```diff
@@ -2349,7 +2349,7 @@ static void _register_variant_builtin_methods_array() {
 	bind_method(Array, append, sarray("value"), varray());
 	bind_method(Array, append_array, sarray("array"), varray());
 	bind_method(Array, resize, sarray("size"), varray());
-	bind_method(Array, insert, sarray("position", "value"), varray());
+	bind_method(Array, insert, sarray("index", "value"), varray());
```

Comment:
I don't think this change is warranted, it's noisy and doesn't fix anything IMO, it technically breaks compatibility though we don't enforce it strongly

Reply:
That's true, it isn't necessarily better than what we had before. Do we guarantee parameter name consistency? 
I thought it would be nice to have the parameters be named the same across functions.

Reply:
We don't guarantee consistency, we prefer it when new methods are added etc. but it's not critical, it'd be more critical between multiple methods in the same class IMO

Reply:
Well, `at_index` is used nowhere else. `index` seems to be the preferred name, although some `Array` functions also use `position`.

Reply:
Ok with me. I'll adjust the PR to not touch the names.